app/views/api/v3/help/show.html.erb

```diff
@@ -10,8 +10,495 @@
   <%= inline_stylesheet 'help.css' %>
   <%= inline_js "standalone/help.js" %>
 </head>
-
-<%= t('api.help.body_html',
+<% if Flipper.enabled?(:yml_help) %>
```

Comment:
If feature flag is enabled, do the new fancy stuff. Else, do the usual HTML locale thing (at the very bottom)